packages/manager/modules/bm-server-components/src/general-info-tile/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,18 @@
+{
+  "server_informations_general": "Informations générales",
```

Comment:
Haven't the translation file for the other languages been moved here?

Reply:
Its not mandatory. I will push to translation and it will be auto populated.